lms/templates/instructor/instructor_dashboard_2/course_info.html

```diff
@@ -5,7 +5,7 @@
 %>
 
 %if settings.FEATURES.get('DISPLAY_ANALYTICS_ENROLLMENTS') or section_data.get('enrollment_message'):
-  <h3 class="hd hd-3">${_("Enrollment Information")}</h3>
+  <h4 class="hd hd-4">${_("Enrollment Information")}</h4>
```

Comment:
That's a good question @cahrens.  My assumption was that the hd-* classes were intended only to be used as overrides.  There are many places in this PR where it looks like they can be safely removed, but we would want to  make sure that without these explicit class declarations, some other styles  up the cascade don't kick in without them.